contracts/programs/access-controller/src/lib.rs

```diff
@@ -5,6 +5,13 @@ use std::mem;
 
 use arrayvec::arrayvec;
 
+#[cfg(feature = "mainnet")]
+declare_id!("DzzjdPWNfwHZmzPVxnmqkkMJraYQQRCpgFZajqkqmU6G");
```

Comment:
@RodrigoAD I pushed a change to allow specifying network via a cargo feature:

`anchor build -- --features mainnet` for mainnet. If no flag specified it will build for localnet instead.

Reply:
Can you plug in the `devnet` ids?

Reply:
Let's not include any devnet or mainnet program IDs, for now.